components/input/input.html

```diff
@@ -0,0 +1,235 @@
+                                      placeholder="Enter Your Password"
+                                    />
+                          
+                                    <button class="btn btn-primary-solid btn-squared">
+                                        <i class="fa-solid fa-arrow-right-to-bracket"></i> Sign up 
+                                    </button>
+                                </form>
+                            </div>
+                            <div class="component-code-section">
+                                <script type="text/plain" class="language-markup code-snippet">
+                                    <form class="form">
+                                        <input
+                                          class="input"
+                                          type="text"
+                                          name="first-name"
+                                          placeholder="Enter Your first name"
+                                        />
+                                        <input
+                                          class="input"
+                                          type="text"
+                                          name="last-name"
+                                          placeholder="Enter Your last name"
+                                        />
+                                        <input
+                                          class="input"
+                                          type="email"
+                                          name="email-id"
+                                          placeholder="Enter Your email id"
+                                        />
+                                        <input
+                                          class="input"
+                                          type="password"
+                                          name="password"
+                                          placeholder="Enter Your Password"
+                                        />
+                              
+                                        <button class="btn btn-primary-solid btn-squared">
+                                            <i class="fa-solid fa-arrow-right-to-bracket"></i> Sign up 
+                                        </button>
+                                    </form>
+                                </script>
+                            </div>
+                        </section>
+
+                        <div class="divider mb-2"></div>
+                        <h2 class="mb-2" id="error-form-input">Error form input</h2>
+                        <section class="component-scetion mb-2">
+                            <div class="component-display">
+                                <form class="form">
+                                    <span class="error-msg">Enter correct email and password</span>
+                                    <input
```

Comment:
you can also add "required" it will check whether the field is empty or not.

Reply:
Ok i will add , thanks for your suggestion.